index.html

```diff
@@ -31,6 +31,155 @@ <h2 class="cover__subtitle">LET’S CONNECT</h2>
         <li><a href="#"><img src="./icons/rabbit.svg" alt="rabbit"></a></li>
       </ul>
     </section>
+    <section class="projects" id="projects">
+      <section class="card1">
+        <section class="card1__area">
+          <div>
+            <img class="card1__image" src="./images/card1.svg" alt="card1 image">
+          </div>
+          <div class="card1__text">
+            <h2 class="card1__text-title">Tonic</h2>
+            <div class="card1__text-list">
+              <h4 class="card1__text-item1">CANOPY</h4>
+              <img class="card1__text-dot" src="./images/dot.png" alt="dot">
+              <h4 class="card1__text-item2">Back End Dev</h4>
+              <img class="card1__text-dot" src="./images/dot.png" alt="dot">
+              <h5 class="card1__text-item2">2015</h5>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          <div class="buttons">
+            <div class="buttons__languages">
+              <div class="buttons__technology buttons__languages-text">html</div>
+              <div class="buttons__technology buttons__languages-text">css</div>
+              <div class="buttons__technology buttons__languages-text">javaScript</div>
+            </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`. Kindly fix that.
- [ ] I suggest you reduce the spacing between the projects as shown in the Figma design.
##### Your Design
![image](https://user-images.githubusercontent.com/29071984/176790807-ae2f7072-622b-4b1a-9612-ca852bfdc855.png)

##### Figma Design
![image](https://user-images.githubusercontent.com/29071984/176790892-b43462b4-87f9-4611-8662-9dd16f648732.png)